src/main/java/ch/njol/skript/classes/data/BukkitEventValues.java

```diff
@@ -1087,6 +1093,63 @@ public Inventory get(final InventoryClickEvent e) {
 				return e.getClickedInventory();
 			}
 		}, 0);
+		// InventoryDragEvent
+		EventValues.registerEventValue(InventoryDragEvent.class, Player.class, new Getter<Player, InventoryDragEvent>() {
+			@Override
+			public @Nullable Player get(InventoryDragEvent event) {
+				return event.getWhoClicked() instanceof Player ? (Player) event.getWhoClicked() : null;
+			}
+		}, EventValues.TIME_NOW);
+		EventValues.registerEventValue(InventoryDragEvent.class, World.class, new Getter<World, InventoryDragEvent>() {
+			@Override
+			public @Nullable World get(InventoryDragEvent event) {
+				return event.getWhoClicked().getWorld();
+			}
+		}, EventValues.TIME_NOW);
+		EventValues.registerEventValue(InventoryDragEvent.class, ItemStack[].class, new Getter<ItemStack[], InventoryDragEvent>() {
+			@Override
+			public @Nullable ItemStack[] get(InventoryDragEvent event) {
+				return event.getNewItems().values().toArray(new ItemStack[0]);
+			}
+		}, EventValues.TIME_NOW);
+		EventValues.registerEventValue(InventoryDragEvent.class, Slot[].class, new Getter<Slot[], InventoryDragEvent>() {
+			@Override
+			public @Nullable Slot[] get(InventoryDragEvent event) {
+				List<Slot> slots = new ArrayList<>(event.getRawSlots().size());
+				InventoryView view = event.getView();
+				for (Integer rawSlot : event.getRawSlots()) {
+					Inventory inventory = view.getInventory(rawSlot);
+					int slot = view.convertSlot(rawSlot);
+					if (inventory == null)
+						continue;
+					if (inventory instanceof PlayerInventory && slot >= 36) {
+						slots.add(new ch.njol.skript.util.slot.EquipmentSlot(((PlayerInventory) view.getBottomInventory()).getHolder(), slot));
+					} else {
+						slots.add(new InventorySlot(inventory, view.convertSlot(rawSlot)));
+					}
+				}
+				return slots.toArray(new Slot[0]);
+			}
+		}, EventValues.TIME_NOW);
+		EventValues.registerEventValue(InventoryDragEvent.class, ClickType.class, new Getter<ClickType, InventoryDragEvent>() {
+			@Override
+			public @Nullable ClickType get(InventoryDragEvent event) {
+				return event.getType() == DragType.EVEN ? ClickType.LEFT : ClickType.RIGHT;
+			}
+		}, EventValues.TIME_NOW);
+		EventValues.registerEventValue(InventoryDragEvent.class, Inventory[].class, new Getter<Inventory[], InventoryDragEvent>() {
+			@Override
+			public @Nullable Inventory[] get(InventoryDragEvent event) {
```

Comment:
```suggestion
			@Override
			@Nullable
			public Inventory[] get(InventoryDragEvent event) {
```
The nullable annotations have to be above the method like so. My IDE automatically formats it to this too, but I correct it.